kinode/src/kernel/mod.rs

```diff
@@ -671,6 +672,7 @@ pub async fn kernel(
     home_directory_path: String,
     contract_address: String,
     runtime_extensions: Vec<(t::ProcessId, t::MessageSender, bool)>,
+    routers: Vec<crate::net::KnsUpdate>,
```

Comment:
need to rename this variable to `bootstrapped_pki_entries` or something clearer. in general, this new code needs to distinguish between "routers" as a concept and what's going on here, which is unrelated to network-routing